enclave_manager/avalon_enclave_manager/avalon_enclave_info.py

```diff
@@ -267,3 +274,70 @@ def create_signup_info(nonce):
 
     # Now we can return the real object
     return signup_info_obj
+
+
+# -----------------------------------------------------------------
+class EnclaveInfo(object):
+    """
```

Comment:
Can we move all the functions inside `EnclaveInfo()` class. May be rename this class to `EnclaveSignupInfo()`